modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java

```diff
@@ -383,6 +384,14 @@ private Price _getPrice(
 			String unitOfMeasureKey)
 		throws Exception {
 
+		if (Validator.isNotNull(unitOfMeasureKey)) {
+			CPInstanceUnitOfMeasure cpInstanceUnitOfMeasure = _cpInstanceUnitOfMeasureLocalService.fetchCPInstanceUnitOfMeasure(cpInstance.getCPInstanceId(), unitOfMeasureKey);
+
+			if (cpInstanceUnitOfMeasure != null) {
+				quantity = cpInstanceUnitOfMeasure.getIncrementalOrderQuantity();
```

Comment:
~~same here calculate the min quantity like the PriceTag~~

No I wrote something wrong, at this point I believe this quantity is the right one so you should not change it with the incrementalOrderQuantity

Reply:
@andrea-ale-sbarra , the quantity is still wrong, but I will try to address it directly in the `.js` that calls the API rather than in the backend.